shell/shell.go

```diff
@@ -47,7 +48,12 @@ func ExecuteShell(uid uint32,
 		cmd.SysProcAttr.Credential = &syscall.Credential{Uid: uid, Gid: gid}
 	}
 
+	if _, err := os.Stat(homeDir); !os.IsNotExist(err) {
+		cmd.Dir = homeDir
+	}
```

Comment:
could we have some fallback in case user has no home directory, or the directory does not exists, e.g.: fall back to "/" ?

Reply:
By default it uses the CWD of the mender-connect process, which is `/` in a "standard installation".
Do you mean hard-coded default, or a configurable one?

Reply:
Let's do it with a hardcoded default to "/"